### DISCUSSION TITLE
Zigbee2mqtt riconnettere dispositivo dopo averlo cancellato

### DISCUSSION THREAD

**Discussion by array81** [indomus/forum#2856] [ZigBee, Z-Wave, Thread, Matter]
Ho un piccolo sensore di movimento zigbee di quelli compatibili con Tuya. Il sensore (modello TS0202) è compatibile con Zigbee2mqtt e lo avevo aggiunto alla mia rete Zigbee senza problemi da diversi mesi. Ultimamente avevo notato che non rilevava più il passaggio di persone, sebbene all'ultima connessione la batteria risultasse ancora al 54% questa di fatto era scarica (ho misurato la tensione). L'ho quindi sostituita e il sensore è tornato ad illuminarsi al mio passaggio ma non comunicava con la mia rete zigbee, quindi ho provato a rimuoverlo da Zigbee2mqtt per poi riconnetterlo. Nel rimuoverlo ho selezionato la rimozione forzata perchè quella normale non funzionava (immagino perchè non c'era comunicazione tra rete e device). Ora il problema è che l'accoppiamento non funziona. Attivo la modalità accoppiamento tramite interfaccia di Zigbee2mqtt, faccio il reset del dispositivo questo per 10/15 ricerca la rete (il led lampeggia velocemente) quindi si spegne e a quel punto torna a illuminarsi a mio passaggio ma Zigbee2mqtt non lo rileva. Ho provato a riavviare Zigbee2mqtt a staccare riattaccare il coordinator e a riflashare quest'ultimo ma niente.

Qualche suggerimento?

**Comment by Scialla** [comment]
Probabilmente, oltre a "forzare" la rimozione hai anche disabilitato il nuovo pairing del dispositivo, sono due opzioni adiacenti e può capitare.
Verifica nel file di configurazione che non sia inserito in blocklist
https://www.zigbee2mqtt.io/guide/configuration/block-pass-list.html

**Reply by array81** [reply]
Grazie della risposta. A dire il vero ho pensato alla cosa e sono abbastanza sicuro di non aver flaggato l'opzione per bloccare il nuovo pairing. Potrei però averlo fatto per sbaglio, appena a casa verifico la cosa. Mi pare di capire che il blocklist sia all'interno del file YAML di configurazione di Zigbee2mqtt, giusto?

**Reply by Scialla** [reply]
Giusto. Non mi vengono in mentre altre possibilità per cui non lo rilevi, sempre che non sia rotto.
Prova anche ad avvicinarlo al coordinator mentre rifai il pairing

**Reply by array81** [reply]
Niente, la sezione "blocklist" nel file di configurazione proprio non c'è.
Vado a rifare il pairing per la 1000 volta in due giorni e mi funziona. Senza che abbia fatto nulla di nuovo semplicemente lo resetto come sempre. E' da venerdì che stavo impazzendo. Comunque a mio avviso il sensore potrebbe avere qualche problema perchè ho veramente provato una marea di volte.

**Reply by Scialla** [reply]
A volte capita; non hai idea di quante volte ho rifatto il pairing del telecomando symphonisk di Ikea, perché non si configurava correttamente..
A volte basta avvicinare il device al coordinator
L'importante è che tu abbia risolto

**Reply by array81** [reply]
Penso sia il sensore ad avere problemi. Il pairing a questo è riuscito ma ora non comunica con la rete. Quando mi rileva si illumina ma Zigbee2MQTT non riceve nulla. Solo se tolgo la batteria e la rimetto, subito dopo c'è un rilevamento da parte di Zigbee2MQTT poi però torna a non ricevere più nulla.

**Reply by Scialla** [reply]
prima di arrenderti, prova ad eliminarlo e a rifare il pairing qualche volta, magari prova anche con batterie diverse e fallo da vicino al router

**Reply by array81** [reply]
Ho fatto qualche test. Di fatto il sensore (che si illumina sempre quando mi rileva) comunica con Zigbee2MQTT solo se lo posiziono attaccato al coordinator. Lo devo mettere a 5/10 cm al massimo, se lo allontano di più, anche se assenza di ostacoli, si illumina ma Zigbee2MQTT non  ottiene informazioni dal sensore. In precedenza era montato a circa 10 metri di distanza con muri nel mezzo e funzionava senza problemi. La batteria ha qualche giorno.
